### PR TITLE
docs(release.md): add blog announcement to post-release tasks

### PR DESCRIPTION
**Pull request by maxgio92**
**What type of PR is this?**

/kind documentation

**Any specific area of the project related to this PR?**

none

**What this PR does / why we need it**:

This PR documents blog announcement as part of post-release tasks.

```release-note
NONE
```

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -106,3 +106,4 @@ Announce the new release to the world!
 
 - Send an announcement to cncf-falco-dev@lists.cncf.io (plain text, please)
 - Let folks in the slack #falco channel know about a new release came out
+- Publish a blog on [Falco website](https://github.com/falcosecurity/falco-website) ([example](https://github.com/falcosecurity/falco-website/blob/master/content/en/blog/falco-0-28-1.md))
```
Comment (leogr) [inline RELEASE.md:109]: Could we put this as the first item?
So one has the blog link already ready when sending the announcements

**Review by leogr — COMMENTED** [review]
I would also add a placeholder to mention the release manager to the release template.
(as we did here https://github.com/falcosecurity/falco/releases/tag/0.28.1)

File: RELEASE.md (hunk 1 above)
Comment (maxgio92) [reply RELEASE.md:109]: Sure! Good idea! Thanks

**Comment by maxgio92** [comment]
> I would also add a placeholder to mention the release manager to the release template.
> (as we did here https://github.com/falcosecurity/falco/releases/tag/0.28.1)

Sure, thanks. Updated accordingly in [#32da3](https://github.com/falcosecurity/falco/pull/1652/commits/32da379153d3be9cef084ac111e7201b86597cd9)

**Review by leogr — APPROVED** [review]
LGTM!

Thank you, Max :)

**Review by leodido — APPROVED** [review]

**Pull request merged**